README.md

```diff
@@ -1,5 +1,13 @@
 # CRA MB Shared UI
```

Comment:
Last thing—I'm becoming less a fan of acronyms these days. It's wordy, but what about something along the lines of:

> Create-React-App Template | Mindbody Shared UI

Reply:
As a reader -- I hear ya. I dropped "Shared" because this template can be used for both SPA's and shared UI's.